source/src/main/java/com/cyb/tms/dao/TmsStoryDAO.java

```diff
@@ -5,12 +5,14 @@
 
 import com.cyb.tms.dto.StoryDTO;
 import com.cyb.tms.dto.SubtaskDTO;
+import com.cyb.tms.dto.TmsSprintDTO;
 import com.cyb.tms.entity.TmsStoryMst;
 
 public interface TmsStoryDAO {
 
 	public long createStory(StoryDTO storyDTO);
 	public long updateStory(StoryDTO storyDTO);
+	public List<String> editStory(StoryDTO storyDTO);

```

Comment:
why you need return type list here?

Method naming conventions should be 
 updateStoryStatus(StoryDTO storyDTO);
editStory(StoryDTO storyDTO);